dist/src/main/distro/bin/mvnd.cmd

```diff
@@ -25,6 +25,9 @@
 @REM   MAVEN_BATCH_PAUSE (Optional) set to 'on' to wait for a key stroke before ending.
 @REM   MAVEN_OPTS        (Optional) Java runtime options used when Maven is executed.
 @REM   MAVEN_SKIP_RC     (Optional) Flag to disable loading of mavenrc files.
+@REM   MVND_ENTRY_FALLBACK (Optional) Flag to disable fallback to pure java mvnd,
+@REM                       default 'true' enable the fallback,
+@REM                       set to 'false' to force execute the native mvnd.

```

Comment:
I wonder if we want to give users a bit more control here. I mean with the proposed implementation users have no way to force the JVM client if they are on natively-supported platform. This might come in handy when debugging. Could we perhaps make it 3-valued, something like 

```
@REM   MVND_CLIENT (Optional) Which mvnd client should be used to communicate with the daemon process.
@REM                       'auto' (default) - prefer the native client mvnd.exe if it is available for the current 
@REM                                                platform and has all dependencies satisfied on the current machine; 
@REM                                                otherwise use the pure Java client
@REM                       'native' - use the native client mvnd.exe
@REM                       'jvm' - use the pure Java client
```

Reply:
new commit switch to the `MVND_CLIENT` with three values.